include/opentracing/dynamic_load.h

```diff
@@ -4,10 +4,11 @@
 #include <opentracing/tracer.h>
 #include <opentracing/tracer_factory.h>
 #include <opentracing/version.h>
+#include <opentracing/config.h>
 #include <system_error>
 
-// OpenTracingMakeTracerFactory provides common hook that can be used to
-// create a TracerFactory from a dynamically loaded library. Users should prefer
+// OpenTracingMakeTracerFactory provides a common hook that can be used to
+// create an TracerFactory from a dynamically loaded library. Users should prefer
```

Comment:
Out of curiosity, why?

Reply:
There are quite a few steps that need to be done if you want to call the OpenTracingMakeTracerFactory function directly, especially if you are doing it with `dlopen` and `dlsym`; furthermore, it's platform-dependent so they would be different on Linux and Windows.

DynamicallyLoadTracingLibrary lets you avoid the manual steps and will work on Windows too (assuming we add support at some point).